charms/worker/k8s/src/token_distributor.py

```diff
@@ -282,45 +346,49 @@ def allocate_tokens(
             token_strategy (TokenStrategy): The strategy of token creation.
             token_type (ClusterTokenType): The type of cluster token.
                 Defaults to ClusterTokenType.NONE.
-
-        Raises:
-            ValueError: If an invalid token_strategy is provided.
         """
-        revoke_on_join = token_strategy == TokenStrategy.CLUSTER
         units = relation.units
         if self.charm.app == relation.app:
             # include self in peer relations
             units |= {self.charm.unit}
         assert relation.app, f"Remote application doesn't exist on {relation.name}"  # nosec
 
         # Select the appropriate token creation strategy
-        token_strat = self.token_creation_strategies.get(token_strategy)
-        if not token_strat:
-            raise ValueError(f"Invalid token_strategy: {token_strategy}")
-
-        log.info("Allocating %s tokens", token_type.value)
-        status.add(ops.MaintenanceStatus(f"Allocating {token_type.value} tokens"))
+        tokenizer = self.token_strategies.get(token_strategy)
+        assert tokenizer, f"Invalid token_strategy: {token_strategy}"  # nosec
+
+        log.info("Allocating %s %s tokens", token_type.name.title(), token_strategy.name.title())
```

Comment:
now this looks better in the logs

|  <!-- --> |  <!-- --> |
| --- | --- |
| before: | `Allocating  tokens` |
| after:  | ```Allocating Control_Plane Cos tokens``` |